README.md

```diff
@@ -52,38 +73,60 @@ async function updateDom() {
   element.appendChild(...);
   element.id = "...";
 
-  // Acquire a lock.
-  await element.displayLock.acquire();
+  // Acquire a lock, and allow it to be committed by the user agent when
+  // necessary when focused, navigated to in find-in-page, anchor links, etc.
+  await element.displayLock.acquire({ activatable: true });
 
   // Append the element to the DOM.
   document.body.appendChild(element);
 
-  // Now we can update the element, causing co-operative updates. After that
-  // resolves, we can commit the element making it visible to the user.
-  element.displayLock.update().then(() => {
+  // Now even if the element is attached to the tree we can do DOM updates,
+  // change style, etc. to it but it won't affect rendering.
+  element.style = "...";
+  element.appendChild(...);
+
+  // Calling update() will allow rendering to go update the style & layout
+  // values through co-operative updates that may last multiple frames.
+  // Note that this will not cause the element to be painted/rendered.
+  // Also, only elements in the locked element's subtree will be updated
+  // co-operatively. If there are other things that needed rendering update
+  // in the document, those things will be updated in one go.
```

Comment:
```suggestion
  // in the document, those things will be updated in one go when the lock is committed.
```

Reply:
Other dirty parts of the document that aren't in a locked subtree will be updated as part of the usual rendering updates, not when the lock gets committed.

I updated the comments around here again